response.js

```diff
@@ -21,6 +21,13 @@ const normalizeHeaders = (req, options) => {
       // do nothing
     }
   }
+  if (req.http && req.http.method) {
```

Comment:
double check this, I think it's req.requestContext

Reply:
Are we sure?  req.requestContext is also a thing in 1.0 but doesn't have the 'http' object.